packages/bundler/src/bundler.ts

```diff
@@ -42,26 +42,26 @@ export interface CadlBundleFile {
 interface PackageJson {
   name: string;
   main: string;
-  cadlMain?: string;
+  typespecMain?: string;
```

Comment:
This will need to be updated based on what we decide for `package.json` format, I think

Reply:
tracked in discussion and comments above.